scripts/ApiParser/ApiParser.ts

```diff
@@ -63,7 +85,7 @@ export abstract class ApiParser {
   public convertToTokenRows(data: ApiResponse): TokenRows {
     const tokens = data.d.data.map((obj) => ({
       tokenName: obj.tokenName,
-      tokenSymbol: obj.tokenSymbol,
+      tokenSymbol: "",
```

Comment:
i found that token symbol is not on the API response and removed it from the type we made. this made it so that i could not just leave it as obj.tokenSymbol. it was essentially a blank string before as well because it is assigned in the filter data step.